point-focus.js

```diff
@@ -1,10 +1,29 @@
-document.addEventListener("DOMContentLoaded", function () {
-    document.body.addEventListener("mousedown", function (evt) {
-        if (!evt.target.setSelectionRange || evt.target.role === 'textbox' || evt.target.hasAttribute("disable-point-focus")) {
-                evt.target.setAttribute("point-focused", true); 
-        }
+document.addEventListener("DOMContentLoaded", function() {
+    var isPointIndicatable = function (el) {
+       var hasTextboxRole = el.getAttribute("role") === "textbox",
+            isDisabled = el.hasAttribute("disable-point-indication"),
+            isTextSelectable = false;
+            try {
+              isTextSelectable = typeof el.selectionStart !== "undefined";
+            } catch (e) {
+              /* nope, it isn't... */
+            }
+            return !isDisabled && (!isTextSelectable && !hasTextboxRole);
```

Comment:
technically yes, but they are conceptually about it being about text as opposed to explicitly disabled, so I thought it was slightly more grokkable with.